grails-app/controllers/org/pih/warehouse/reporting/ReportController.groovy

```diff
@@ -415,7 +415,7 @@ class ReportController {
                     binLocations = binLocations.findAll { it.status == params.status }
                 }
 
-                String csv = ReportUtil.getCsvForListOfMapEntries(binLocations, binLocationCsvHeader, binLocationCsvRow)
+                String csv = ReportUtil.getCsvForListOfMapEntries(binLocations, this.&binLocationCsvHeader, this.&binLocationCsvRow)
```

Comment:
What does `this.&` do in that context?

Reply:
it's a  [method pointer](https://groovy-lang.org/operators.html#method-pointer-operator)
From what I understand if you want to pass a class method as argument in newer versions of groovy you have to pass it as a reference pointer